api/helpers/jwt.js

```
@@ -17,6 +18,18 @@ async function generateToken(user) {
      return error.message;
   }
 }
+
+async function decodeToken(token) {
+  try {
+    const decoded = jwt.verify(token, JWTSecret)
+    const user = await User.findById(decoded.subject).exec();
+    if(!user) throw Error('no such user');
+    return user;
+  } catch (error) {
+    return null
```

Wouldn't it be better to return the error message

---

If we resolve the error here when using this helper method somewhere else we will not get to the catch block

---

Okay